### PR TITLE
mstpd: bump version to 0.1.0

### PR DESCRIPTION
**Pull request by rubensfig**

### CODE REVIEW

File: recipes-support/mstpd/mstpd_git.bb [hunk 1]
```diff
@@ -5,14 +5,12 @@ LIC_FILES_CHKSUM = "file://LICENSE;md5=4325afd396febcb659c36b49533135d4"
 
 SRC_URI = "\
   git://github.com/mstpd/mstpd.git;protocol=https \
-  file://0001-libnetlink-introduce-rta_nest-and-u8-u16-u64-helpers.patch \
-  file://0002-libnetlink-Set-NLA_F_NESTED-in-rta_nest.patch \
```
Comment (KanjiMonster) [inline recipes-support/mstpd/mstpd_git.bb:9]: Why are you removing these lines? Please explain in your commit message.

Also please delete the files itself.

File: recipes-support/mstpd/mstpd_git.bb [hunk 2]
```diff
@@ -5,14 +5,12 @@ LIC_FILES_CHKSUM = "file://LICENSE;md5=4325afd396febcb659c36b49533135d4"
...
   file://0003-libnetlink-Add-helper-to-add-a-group-via-setsockopt.patch \
   file://0004-WIP-set-CIST-MSTI-port-states-to-the-newly-added-per.patch \
```
Comment (KanjiMonster) [inline recipes-support/mstpd/mstpd_git.bb:9]: And it might make sense to rename these to 0001/0002 then.

**Review by KanjiMonster — CHANGES_REQUESTED** [review]

**Comment by KanjiMonster** [comment]
Since we are targeting the 0.1.0 release tag, I think we should rename the recipe to have an actual version instead of just _git

**Review by KanjiMonster — CHANGES_REQUESTED** [review]
With the mstpd_0.1.0.bb there is no reason for the mstpd_git to exist, so please drop it.

File: recipes-support/mstpd/mstpd_0.1.0.bb [hunk 3]
```diff
@@ -5,14 +5,12 @@ LIC_FILES_CHKSUM = "file://LICENSE;md5=4325afd396febcb659c36b49533135d4"
...
-  file://0002-libnetlink-Set-NLA_F_NESTED-in-rta_nest.patch \
-  file://0003-libnetlink-Add-helper-to-add-a-group-via-setsockopt.patch \
-  file://0004-WIP-set-CIST-MSTI-port-states-to-the-newly-added-per.patch \
+  file://0001-libnetlink-Add-helper-to-add-a-group-via-setsockopt.patch \
+  file://0002-WIP-set-CIST-MSTI-port-states-to-the-newly-added-per.patch \
   file://bridge-stp.conf"
 
-PV = "0.0.8+git${SRCPV}"
-SRCREV = "4d98707b9047a74bc473345875e60ca5a24f0c96"
+PV = "0.1.0+git${SRCPV}"
```
Comment (jklare) [inline recipes-support/mstpd/mstpd_0.1.0.bb:12]:
```suggestion
PV = "0.1.0+bisdn1"
```

**Pull request closed without merging**